src/main/java/io/cdap/plugin/kudu/KuduSink.java

```diff
@@ -106,7 +106,7 @@ public void configurePipeline(PipelineConfigurer configurer) {
     if (kuduSinkConfig.containsMacro("master") || kuduSinkConfig.containsMacro("name")) {
       return;
     }
-    createKuduTable();
+    createKuduTable(failureCollector);
```

Comment:
this was existing bad behavior, but nothing should ever be created in configurePipeline(). The method is not meant to have any side effects, since it can be called many times in different contexts. Please remove this call.

Reply:
Done.